tool/tsh/main.go

```diff
@@ -383,24 +393,34 @@ func makeClient(cf *CLIConf) (tc *client.TeleportClient, err error) {
 		return nil, err
 	}
 
-	// prep client config:
-	c := &client.Config{
-		Stdout:             os.Stdout,
-		Stderr:             os.Stderr,
-		Stdin:              os.Stdin,
-		Username:           cf.Username,
-		ProxyHost:          cf.Proxy,
-		Host:               cf.UserHost,
-		HostPort:           int(cf.NodePort),
-		HostLogin:          hostLogin,
-		Labels:             labels,
-		KeyTTL:             time.Minute * time.Duration(cf.MinsToLive),
-		InsecureSkipVerify: cf.InsecureSkipVerify,
-		LocalForwardPorts:  fPorts,
-		ConnectorID:        cf.ExternalAuth,
-		SiteName:           cf.SiteName,
-		Interactive:        cf.Interactive,
-	}
+	// 1: start with the defaults
+	c := client.MakeDefaultConfig()
+
+	// 2: override with `./tsh` profiles
+	if err = c.LoadProfile(""); err != nil {
+		fmt.Printf("WARNING: Failed loading tsh profile.\n%v\n", err)
```

Comment:
because failing doesn't make sense (user may have supplied enough CLI flags to proceed) and we don't want to hide the error.